content/chainguard/chainguard-images/staying-secure/fedramp-considerations/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "FedRAMP Technical Considerations & Risk Factors"
+linktitle: "FedRAMP Considerations"
+type: "article"
+description: "A conceptual overview of Chainguard FIPS Images."
+date: 2025-01-29T15:56:52-07:00
+lastmod: 2025-01-29T15:56:52-07:00
+draft: false
+tags: ["IMAGES", "PRODUCT", "CONCEPTUAL", "FIPS"]
+images: []
+menu:
+  docs:
+    parent: "features"
+weight: 007
+toc: true
+---
+
+Many frequently asked questions revolve around how organizations are meant to stay on top of the changing landscape for FedRAMP, PMOS, Revisions, and Certificates. This article outlines various considerations and risk factors organizations should keep in mind when working to become and stay FedRAMP authorized.
+
+## Important Considerations for PMO Revision Trends
+
+There are a number of things one should keep in mind when analyzing revision trends from the FedRAMP Program Management Office (PMO) — which oversees the development of the FedRAMP program — and the changes in [FIPS 140-3](https://csrc.nist.gov/projects/fips-140-3-transition-effort). The following are of particular importance:
+
+* FedRAMP authorization has tended to only include external customer communication under scope. Now, though, the FedRAMP PMO is expecting both external and internal communications.
```

Comment:
```suggestion
* FedRAMP authorization has tended to only include external customer communication under scope. Now, though, the FedRAMP PMO is expecting both external and internal communications in scope.
```

Again, just thinking of the not-so-good reader trying to grok and only doing so slowly unless we are ultra annoyingly specific.